README.md

```diff
@@ -15,8 +15,13 @@ For an overview of the whole Add-on Store, read [the design overview](./docs/des
 ### About security 
 Ensuring that an add-on is safe to run is a difficult challenge that isn't addressed here.
 However, the metadata for a new submission (add-on release) can be confirmed to match its manifest description.
```

Comment:
thoughts on removing these 2 sentences and replacing it with something like?
It seems like we are trying to tackle the first sentence now, and the second sentence doesn't really seem relevant to security.

>Add-ons are run at user's own risk, add-ons in the add-on store do not undergo human security audits.
>The add-on store includes the following security measures:

Reply:
any thoughts on this change?